Src/WitsmlExplorer.Frontend/components/GlobalStyles.tsx

```diff
@@ -97,6 +97,12 @@ const GlobalStyles = createGlobalStyle`
       }
     }
   }
+
+  .serversList {
+    tr:nth-child(odd) {
+       background-color: ${colors.interactive.tableHeaderFillResting};
```

Comment:
This style makes it so that only every other row in the server manager becomes gray on hover. Was that the intention?

Reply:
@janburak  based on below design specifications in figma we have to provide alternate colors to the table

![image](https://user-images.githubusercontent.com/26452934/210568126-b9d3d97d-8cea-44ca-81ba-f4c5d6c0cfa1.png)

Reply:
The style does match the design when not mousing over the rows. Does the design specify on hover colors? If not, then adding 
` tr:nth-child(even) {
      background-color: white;
    }`
would ensure that the rows are the expected color even when mousing over.